crates/common/src/opa.rs

```diff
@@ -30,6 +36,12 @@ pub enum PolicyLoaderError {
 
     #[error("Error loading OPA policy: {0}")]
     SawtoothCommunicationError(#[from] SawtoothCommunicationError),
+
+    #[error("URL request error: {0}")]
+    UrlRequestError(#[from] reqwest::Error),
+
+    #[error("URL IO error: {0}")]
+    UrlIoError(#[from] std::io::Error),
```

Comment:
No objection but it's worth being aware that if someone adds further code to this crate using suffix `?` for some *other* source of IO errors then it may surprise them to find it being reported as being URL-related.